NEWS.md

```diff
@@ -1,7 +1,7 @@
 # OpenDDS Releases
 
-## Version 3.14 of OpenDDS
-Released %TIMESTAMP%
+## Version 3.14.0 of OpenDDS
```

Comment:
```suggestion
## Version 3.14 of OpenDDS
```

Reply:
I think this was a byproduct of me rewriting the code that processes the version to use. When do you want the full form of the version used?

Reply:
I'm just trying to keep it consistent with what we have now.  I think that is when the 3rd number is a 0, the trailing `.0` is omitted.

Reply:
I think that's kinda ambiguous, but I will change it back.